modules/n1ql/examples/transactions/results.jsonc

```diff
@@ -48,6 +48,7 @@
       "results": []
     }
   },
+  // tag::user0[]
```

Comment:
Stash these changes and reuse them to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)